src/events/reaction_add.rs

```diff
@@ -36,5 +38,28 @@ pub async fn reaction_add(ctx: client::Context, event: Reaction) -> Result<()> {
             }
         }
     }
+    handle_emoji_logging(ctx, event).await?;
+    Ok(())
+}
+
+async fn handle_emoji_logging(ctx: client::Context, event: Reaction) -> Result<()> {
+    let id = if let Custom { id, .. } = event.emoji {
+        id
+    } else {
+        return Ok(());
+    };
```

Comment:
minor style thing, but in most other places i've been using a match for this idiom, rather than a if let, so it'd be good to keep that consistent (literally the same thing, but match rather than `if let`